composer/lib/dependabot/composer/update_checker/version_resolver.rb

```diff
@@ -320,6 +320,9 @@ def handle_composer_errors(error)
           elsif error.message.start_with?("Allowed memory size") ||
                 error.message.start_with?("Out of memory")
             raise Dependabot::OutOfMemory
+          elsif error.error_context[:process_termsig] == 9
```

Comment:
Nit: could we add this to constant

```suggestion
          elsif error.error_context[:process_termsig] == Dependabot::SharedHelpers::SIGKILL
```

Reply:
Noice! I was hopeful there might be a constant in the stdlib that I was failing to google, a la https://golang.org/pkg/syscall/ . I'll update to declare our own.